config/coordinator/coordinator-local-dev.config-traces-v2.overrides.toml

```diff
@@ -1,13 +1,13 @@
 [prover]
 [prover.execution]
-fs-requests-directory = "/data/prover/v3/execution/requests"
-fs-responses-directory = "/data/prover/v3/execution/responses"
+fs-requests-directory = "tmp/local/prover/v2/execution/requests"
```

Comment:
I thinks we still needs v3 instead of v2 here, because of the`prover-v3` we use in "traces-v2", see the config: https://github.com/ConsenSys/zkevm-monorepo/blob/3822-rejected-transaction-api-service/docker/config/prover/v3/prover-config.toml#L11

Reply:
Good catch, it's my mistake